Sources/ComposableArchitecture/Store.swift

```diff
@@ -5,7 +5,81 @@ import Foundation
 /// around to views that need to interact with the application.
 ///
 /// You will typically construct a single one of these at the root of your application, and then use
-/// the ``scope(state:action:)-9iai9`` method to derive more focused stores that can be passed to subviews.
+/// the ``scope(state:action:)-9iai9`` method to derive more focused stores that can be passed to subviews:
+///
+/// ```swift
+/// @main
+/// struct MyApp: App {
+///   var body: some Scene {
+///     WindowGroup {
+///       RootView(
+///         store: Store(
+///           initialState: AppState(),
+///           reducer: appReducer,
+///           environment: AppEnvironment(
+///             ...
+///           )
+///         )
+///       )
+///     }
+///   }
+/// }
+/// ```
+///
+/// ### Scoping
+///
+/// The most important operation defined on ``Store`` is the ``scope(state:action:)-9iai9`` method, which allows you to transform a store into one that deals with local state and actions. This is necessary for passing stores to subviews that only care about a small portion of the entire application's domain.
```

Comment:
Did we wanna keep docs at 110 cols before merging?

Reply:
won't swift-format take care of this for us? if not I will fix it.